### DISCUSSION TITLE
步骤条使用的时候图标变成了白框

### DISCUSSION THREAD

**Discussion by Tesla-colder** [TaleLin/lin-ui#1203] [求助] [answered]
![屏幕截图 2020-12-22 195852](https://user-images.githubusercontent.com/63287116/102886509-329c1000-4490-11eb-9105-223788f73e37.jpg)
![屏幕截图 2020-12-22 195915](https://user-images.githubusercontent.com/63287116/102886517-34fe6a00-4490-11eb-86cd-3ff8903737ee.jpg)
本人小白，请大神指点

**Comment by murongg** [comment]
请提供[微信小程序代码片段](https://developers.weixin.qq.com/miniprogram/dev/devtools/minicode.html)供我们测试

**Reply by Tesla-colder** [reply]
我现在是学生，才开始学，用的微信云数据库，可以测试吗

**Reply by Tesla-colder** [reply]
代码片段：https://developers.weixin.qq.com/s/B1TFhNmE7zmn

**Comment by juzi214032** [comment, marked as answer]
提供的代码片段无法运行，直接发一下你的步骤条图标的图片呢，看看是不是图片的问题

**Reply by Tesla-colder** [reply]
没自己设置图片，就用的基础的

**Reply by Tesla-colder** [reply]
有没有可能是我基础库或环境啥的配置有问题

**Reply by juzi214032** [reply]
基础库是用的最新吗

**Reply by juzi214032** [reply]
我这边无法复现你的问题，建议再自行排查一下

**Reply by Tesla-colder** [reply]
好的，谢谢了
